packages/api/src/app.ts

```diff
@@ -0,0 +1,85 @@
+
+import http from 'http';
+import { format } from 'url';
+import { initMongoNoIndexes, isMongoDBHealthy } from '@sorry-cypress/mongo';
+import stoppable from 'stoppable';
+import { ApolloServer } from 'apollo-server-express';
+import {
+  ApolloServerPluginLandingPageGraphQLPlayground,
+  ApolloServerPluginLandingPageDisabled,
+} from 'apollo-server-core';
+import express from 'express';
+
+import { InstancesAPI } from './datasources/instances';
+import { ProjectsAPI } from './datasources/projects';
+import { RunsAPI } from './datasources/runs';
+import { RunTimeoutAPI } from './datasources/runTimeout';
+import { SpecsAPI } from './datasources/specs';
+import { resolvers } from './resolvers';
+import { typeDefs } from './schema/schema';
+import { catchRequestHandlerErrors } from './lib/express';
+
+
+
+export const start = async function start(host: string, port: number, basePath: string, apolloPlayground: string) {
+  await initMongoNoIndexes();
+  const dataSources = {
+    runsAPI: new RunsAPI(),
+    instancesAPI: new InstancesAPI(),
+    projectsAPI: new ProjectsAPI(),
+    specsAPI: new SpecsAPI(),
+    runTimeoutAPI: new RunTimeoutAPI(),
+  };
+
+  const apolloServer = new ApolloServer({
+    typeDefs,
+    resolvers,
+    dataSources: () => dataSources,
+    introspection: true,
+    plugins: [
+      apolloPlayground === 'false'
+        ? ApolloServerPluginLandingPageDisabled()
+        : ApolloServerPluginLandingPageGraphQLPlayground(),
+    ],
+  });
+
+
+  const app = express();
+
+  app.disable('x-powered-by');
```

Comment:
Maybe we can use https://www.npmjs.com/package/helmet instead of only removing the powered-by header?

Reply:
Good candidate for a separate enhancement